data/converters/converters.go

```diff
@@ -0,0 +1,194 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+// Int64NOOP .....
+var Int64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeInt64,
+}
+
+// BoolNOOP .....
+var BoolNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeBool,
+}
+
+// Float64NOOP .....
+var Float64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeFloat64,
+}
+
+// StringNOOP value is already in the proper format
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToOptionalString any value as a string
+var AnyToOptionalString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		str := fmt.Sprintf("%+v", v) // the +v adds field names
+		return &str, nil
+	},
+}
+
+// AnyToString any value as a string
+var AnyToString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		return fmt.Sprintf("%+v", v), nil // the +v adds field names
+	},
+}
+
+// Float64ToOptionalFloat64 optional float value
+var Float64ToOptionalFloat64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableFloat64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(float64)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[float] expected float64 input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// Int64ToOptionalInt64 optional int value
+var Int64ToOptionalInt64 = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableInt64,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		val, ok := v.(int64)
+		if !ok { // or return some default value instead of erroring
+			return nil, fmt.Errorf("[int] expected int64 input but got type %T", v)
+		}
+		return &val, nil
+	},
+}
+
+// UInt64ToOptionalUInt64 optional int value
+var UInt64ToOptionalUInt64 = data.FieldConverter{
```

Comment:
Should be cased `Uint64`, as per the corresponding field type, and following the pattern of the standard library `strconv` pkg https://golang.org/pkg/strconv/